### PR TITLE
QS + analysis

### PR DESCRIPTION
**Pull request by blueur**
- add multiple load and displacement selection #120 
- add output charts in analysis

### CODE REVIEW

File: backend/preprocessing/tst_data_lib.py [hunk 1]
```diff
@@ -1290,7 +1291,11 @@ def _save_preprocessed_measures(self):
         self.logger.info("Save measures")
         for measures in self.measures_list:
             with self.logger.indent:
-                measures["df"].to_csv(measures["preprocessed_fp"], index=False)
+                measures["df"].to_csv(
+                    measures["preprocessed_fp"],
+                    index=False,
+                    float_format="%.16g",
```
Comment (sbancal) [inline backend/preprocessing/tst_data_lib.py:1297]: Running on Ubuntu 22.04 -> different values starting at least from `"%.12g"

File: backend/preprocessing/hysteresis_loops.py [hunk 2]
```diff
@@ -159,6 +159,7 @@ def main():
                             hys_fp,
                         ),
                         index=False,
+                        float_format="%.16g",
```
Comment (sbancal) [inline backend/preprocessing/hysteresis_loops.py:162]: same: `"%.12g"`

**Comment by sbancal** [comment]
... see  a8d35a4b3328363bebd9f238e08826f39d15e1fc

**Review by sbancal — CHANGES_REQUESTED** [review]
Please check to avoid 100% empty graphs like on 
http://localhost:8080/fatigue_database/tests_dashboard?exp=5&tests=30&tests=33

thks : )

**Review by sbancal — APPROVED** [review]
merci : )

**Comment by vsdharun** [comment]
@sbancal It seems the x and y-axis values are swaped. could you fix it ?



image

**Pull request merged**